pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -215,9 +202,18 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		opt.StartTime = start
 	}
 
+	timeout := opt.Timeout
+	if timeout == 0 {
```

Comment:
Can someone double check my understanding of this double timeout == 0 check?
It seems like the logic is essentially:
if opt.Timeout == 0:
   if suite.TestTimeout == 0:
       timeout = 15*time.Minute
  else: timeout = suite.TestTimeout
else:
  timeout = opt.Timeout

Reply:
I think so.  

We're picking the first non-zero value from this:

- opt.Timeout
- suite.TestTimeout
- 15 minutes